src/main/webapp/js/date.js

```diff
@@ -321,4 +321,30 @@ function parseDate(val) {
 		}
 	}
 	return null;
-}
\ No newline at end of file
+}
+
+/**
+ * Returns the timezone (in string) from a parsed date. The date is parsed in such a way
+ * that, when the date string is split for every whitespace encountered, the timezone string
+ * is in the last index of the splitted string.
+ */
+function getTimeZoneStringFromDate(date) {
+    var currentDateInString = date.toString().split(' ');
+    var userTimeZone = currentDateInString[currentDateInString.length - 1];
+    return userTimeZone;
+}
```

Comment:
Header comments shouldn't go into _how_ things work. It should be inside the method body on the relevant line (e.g before the operation).
Also the comment is not so clear; it implies that "when split, the timezone is in the last index", but which part of the operation shows that? Perhaps what you're trying to (and should) say is that js date object has the format "... ... (timezone)" and thus after split it will be on the last index.